jottacloud: integration tests failing with read/write mime type

I recently put in a new integration test to make sure that when backends advertise that they can Read or Write MimeTypes that they actually can. See dfadd9896999813397cf7457c01ffcc789162a56

Most of the integration tests were easy to fix up, but Jottacloud is harder!

Jottacloud can apparently read MimeTypes but it isn't setting them. It is advertising them as settable though.

```
~/go/src/github.com/rclone/rclone/backend/jottacloud$ go test -v -run TestIntegration/FsMkdir/FsPutFiles/ObjectMimeType
=== RUN   TestIntegration
    fstests.go:396: Using remote "TestJottacloud:"
=== RUN   TestIntegration/FsMkdir
=== RUN   TestIntegration/FsMkdir/FsPutFiles
=== RUN   TestIntegration/FsMkdir/FsPutFiles/ObjectMimeType
    fstests.go:1303: 
        	Error Trace:	fstests.go:1303
        	Error:      	Not equal: 
        	            	expected: "text/csv"
        	            	actual  : "text/plain"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-text/csv
        	            	+text/plain
        	Test:       	TestIntegration/FsMkdir/FsPutFiles/ObjectMimeType
        	Messages:   	can read and write mime types but failed
--- FAIL: TestIntegration (5.18s)
    --- FAIL: TestIntegration/FsMkdir (4.55s)
        --- FAIL: TestIntegration/FsMkdir/FsPutFiles (3.01s)
            --- FAIL: TestIntegration/FsMkdir/FsPutFiles/ObjectMimeType (0.08s)
FAIL
exit status 1
FAIL	github.com/rclone/rclone/backend/jottacloud	5.184s
```

In `Update` it is never calling `fs.MimeType(src)` which is why the mime type never gets set.

To fix this we could either

1. Set `WriteMimeType` to `false` indicating the jottacloud backend can't set MimeTypes
2. Work out where to supply the MimeType/ContentType in `Update`.

@buengese and @albertony any ideas here?

--- a/backend/jottacloud/jottacloud.go
+++ b/backend/jottacloud/jottacloud.go
@@ -717,7 +717,7 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 		CaseInsensitive:         true,
 		CanHaveEmptyDirectories: true,
 		ReadMimeType:            true,
-		WriteMimeType:           true,
+		WriteMimeType:           false,
 	}).Fill(ctx, f)
 	f.srv.SetErrorHandler(errorHandler)
 	if opt.TrashedOnly { // we cannot support showing Trashed Files when using ListR right now

--- a/docs/content/overview.md
+++ b/docs/content/overview.md
@@ -30,7 +30,7 @@ Here is an overview of the major features of each cloud storage system.
 | Google Photos                | -           | No      | No               | Yes             | R         |
 | HTTP                         | -           | No      | No               | No              | R         |
 | Hubic                        | MD5         | Yes     | No               | No              | R/W       |
-| Jottacloud                   | MD5         | Yes     | Yes              | No              | R/W       |
+| Jottacloud                   | MD5         | Yes     | Yes              | No              | R         |
 | Koofr                        | MD5         | No      | Yes              | No              | -         |
 | Mail.ru Cloud                | Mailru ⁶    | Yes     | Yes              | No              | -         |
 | Mega                         | -           | No      | No               | Yes             | -         |

